app/Services/Account/Activity/Activity/CreateActivity.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Services\Account\Activity\Activity;
+
+use App\Services\BaseService;
+use App\Models\Account\Activity;
+use App\Models\Account\ActivityType;
+use App\Models\Journal\JournalEntry;
+use App\Models\Instance\Emotion\Emotion;
+
+class CreateActivity extends BaseService
+{
+    /**
+     * Get the validation rules that apply to the service.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'account_id' => 'required|integer|exists:accounts,id',
+            'activity_type_id' => 'nullable|integer',
+            'summary' => 'required|string:255',
+            'description' => 'nullable|string:400000000',
+            'date' => 'required|date_format:Y-m-d',
```

Comment:
You may add `date` as a format
```suggestion
            'date' => 'required|date|date_format:Y-m-d',
```

Reply:
Is it necessary though?

Reply:
No I don't think so either